Modal.astro

```diff
@@ -9,7 +9,7 @@ interface Props {
 const { triggerId, title, closeText = 'Close' } = Astro.props
 ---
 
-<div class="modal" role="dialog" aria-labelledby={triggerId}>
+<dialog class="modal" role="dialog" aria-labelledby={triggerId}>
```

Comment:
@david-abell since we now use the default `<dialog>` element we can omit `role="dialog"` 🙂